pkg/resource/deploy/providers/registry.go

```diff
@@ -316,21 +317,15 @@ func (r *Registry) Diff(urn resource.URN, id resource.ID, olds, news resource.Pr
 		// (which we should have loaded during diff search), and we will not unload it.
 		provider, ok = r.GetProvider(mustNewReference(urn, id))
 		contract.Assertf(ok, "Provider must have been registered at some point for DBR Diff (%v::%v)", urn, id)
-
-		diff, err := provider.DiffConfig(urn, olds, news, allowUnknowns, ignoreChanges)
-		if err != nil {
-			return plugin.DiffResult{Changes: plugin.DiffUnknown}, err
```

Comment:
Is this result no longer needed?

Reply:
It just falls through into doing this below + handling the default diffing we do for resources. We could add this back if we're worried that auto-diff on provider config might cause issues.

Reply:
Oh, I didn't see that this was identical to the error handling below. I think this should be fine.